warehouse/query-core/src/main/java/datawave/query/jexl/visitors/whindex/WhindexVisitor.java

```diff
@@ -695,17 +702,15 @@ private JexlNode visitLeafNode(JexlNode node, ExpandData eData) {
             else if (mappableFields.contains(leafEntry.getKey())) {
                 // if it's a mappable field, do we have the term/value match needed to map it?
                 List<Map.Entry<String,JexlNode>> fieldValueMatches = new ArrayList<>();
-                remainingLeafNodes.entries().stream()
-                                .filter(x -> x.getValue() instanceof ASTEQNode && isFieldValueMatch((ASTEQNode) x.getValue(), leafEntry.getKey()))
-                                .forEach(fieldValueMatches::add);
-                remainingAndedNodes.entries().stream()
-                                .filter(x -> x.getValue() instanceof ASTEQNode && isFieldValueMatch((ASTEQNode) x.getValue(), leafEntry.getKey()))
-                                .forEach(fieldValueMatches::add);
+                remainingLeafNodes.entries().stream().filter(x -> isFieldValueMatch(x.getValue(), leafEntry.getKey())).forEach(fieldValueMatches::add);
+                remainingAndedNodes.entries().stream().filter(x -> isFieldValueMatch(x.getValue(), leafEntry.getKey())).forEach(fieldValueMatches::add);
                 
                 for (Map.Entry<String,JexlNode> fieldValueMatch : fieldValueMatches) {
                     String value = getStringValue(fieldValueMatch.getValue());
                     
                     if (value != null) {
+                        value = value.toLowerCase();
```

Comment:
Push this normalization into `getStringValue`?

Reply:
That's where I originally had it.  I thought it would be more explicit to have it here.  My concern was that someone using the getStringValue method in the future might not know that it converts to lowercase.

Reply:
Works for me